assets/js/translator.js

```diff
@@ -73,7 +73,9 @@ if(modeEnabled('translation')) {
             refreshLangList(true);
             muteLanguages();
             localizeInterface();
-            translateText();
+            if(this.id !== 'detect') {
```

Comment:
Can this particular conditional wrap all the function calls to prevent unnecessary work?